inst/tests/tests.Rraw

```diff
@@ -18278,3 +18278,13 @@ test(2245.1, dt[1], data.table(foo = 1L, bar = 4L)) # Single index should be int
 test(2245.2, copy(dt[1]), data.table(foo = 1:3)) # Revert to data.frame syntax: Interpret index as column
 rm(.datatable.aware)
 test(2245.3, dt[1], data.table(foo = 1L, bar = 4L)) # Default in this environment should be datatable-aware
+
+# 5392 split(x,f) does not work with formula f
```

Comment:
Generally we don't want to describe the bug in the comment here. In 10 years we won't remember what behavior didn't work "back then"; instead it'll be clearer to read a comment about what this test is trying to guarantee.

```suggestion
# 5392 split(x,f) works with formula f
```